docs/my-website/docs/providers/aiml.md

```diff
@@ -0,0 +1,144 @@
+# AI/ML API
```

Comment:
need to add this to the `sidebar.js` for this to show up on the docs @waterstark